modules/simple-payments/simple-payments.php

```diff
@@ -54,6 +54,12 @@ public function init_hook_action() {
 		$this->setup_cpts();
 
 		add_filter( 'the_content', array( $this, 'remove_auto_paragraph_from_product_description' ), 0 );
+
+		$is_available = $this->is_enabled_jetpack_simple_payments();
+		jetpack_register_block( 'simple-payment', array(), array(
+			'available' => $is_available,
+			'unavailable_reason' => ( ! $is_available ? 'missing_plan' : '' ),
+		) );
```

Comment:
we should also do a check for connection status here

Reply:
Since this is a shared wordpress.com file, we'll need to ensure that `jetpack_register_block` is also declared on wordpress.com